script.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable linebreak-style */
+class Books {
+  constructor(title = '', author = '') {
+    this.title = title;
+    this.author = author;
+    this.books = localStorage.getItem('books') !== null
+      ? JSON.parse(localStorage.getItem('books'))
+      : '';
+  }
+
+  saveBook(BTSave = this.books) {
+    const storeBook = JSON.stringify(BTSave);
+    localStorage.setItem('books', storeBook);
+  }
+
+  addBook() {
+    if (this.books === '' || this.books.length <= 0) {
+      this.books = [
+        {
+          Id: 1,
+          Btitle: this.title,
+          Bauthor: this.author,
+        },
+      ];
+      this.saveBook();
+    } else {
+      const lastId = this.books[this.books.length - 1].Id + 1;
+      const book = {
+        Id: lastId,
+        Btitle: this.title,
+        Bauthor: this.author,
+      };
+      this.books.push(book);
```

Comment:
# Optional:
- Please use an array method that doesn't mutate the array: Use [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), instead of `push`